### ISSUE TITLE
Is the test DB available?

### ISSUE THREAD

**Issue by yakovkeselman** [uber/sql-differential-privacy#8]
Is the "test" DB available for download? I was not able to find it in the repository. Thanks.

**Comment by duanbing** [comment]
Yes if you mean "test" database for the example, have a look at `./src/test/resources/schema.yaml`.
